[Vigilitas Interstellar] Adds the Shrike-class Fast Response Frigate

Burning02 opened the pull request:
## About The Pull Request

Adds the Shrike-class Fast Response Frigate, a Vigilitas Interstellar vessel centered around combat (Of the PvE and PvP variety), and security contracting as a vessel of the largest security contractor in the Galaxy, with 2 new VI outfits (Captain, and 2IC (Sergeant for this vessel)). Weaponry currently present within are placeholders until the Sharplite rework is merged.

***
>_Description is currently a WIP_
><img width="574" height="766" alt="image" src="https://github.com/user-attachments/assets/c0ebf8b7-b9e5-41fc-91cf-317694ec547e" />
<details>
  <summary>Strongdmm photos</summary>
  
  <p align="center">Areas:</p>
<img width="1152" height="1120" alt="image" src="https://github.com/user-attachments/assets/ffd09854-eccc-4f22-9695-572089bb933c" />

  <p align="center">Without Areas:</p>
<img width="1152" height="1120" alt="image" src="https://github.com/user-attachments/assets/58e92054-a303-4dd5-921d-6cca1fa5fdab" />
</details>

  


## Why It's Good For The Game

Whilst Vigilitas may already have a combat related vessel ingame, it is unfortunately locked to the Loss Prevention department of VI, instead of a more broad generalist vessel. This addition changes that, giving VI players who may have wanted to play the faction outside of Nanotrasen and N + S Shipping vessel, without having to make a character who may not potentially fit with the LP department.

## Changelog

:cl:
add: Adds the Shrike-class Fast Response Frigate, a new Vigilitas Interstellar vessel.
add: Adds two additional VI outfits.
add: New secure safe subtype
/:cl:

Burning02 committed:
Yeah

Burning02 committed:
Update nanotrasen.dm

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Update nanotrasen_shrike.json

Burning02 committed:
Update nanotrasen_shrike.json

Burning02 committed:
E

Burning02 committed:
I love coding

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Some very small and random things

Re-did the dorms decaling, and added food (The crew hungers)

Erikafox commented on _maps/configs/nanotrasen_shrike.json:
```diff
@@ -0,0 +1,47 @@
+		"WEAPONS"
+	],
+	"map_short_name": "Shrike-class",
+	"description": "A vessel fresh off of Nanotrasen Shipwork yards, the Shrike-class is purpose-built for Vigilatis Interstellar fast response times. Whilst the hull may not boast a grand armament, its small size allows the fast paced construction of it nonetheless, which has allowed its quick adoption by VI forces.",
```
```suggestion
	"description": "A new class of vessel fresh from Nanotrasen Shipworks, the Shrike-class is purpose-built for Vigilitas Interstellar's Fast Response Teams. The hull is considered under-armed by most people, which is an acceptable trade for Vigilitas as it narrowly circumvents several Core World treaties on militarized private assets, reducing the legal overhead of manufacturing. The small size also allows rapid deployment into conflict areas. The designs strengths have lead to its quick adoption by Vigilitas for both Core and Frontier operations.",
```

Erikafox reviewed (dismissed):
This ship's biggest issue is that you have way too much dead space. There'll be a few ideas in here. You're going to need to reduce the amount of hallway and things up against walls for proper work areas, smaller offices, a nicer eating area, so forth. Don't be afraid to rip up rooms that are already present.
<br>
<img width="927" height="467" alt="image" src="https://github.com/user-attachments/assets/4d5af72b-e66d-427c-9754-cede570ea4ae" />
Safes do not grab contents on their tile. If you want something like this, you either need to subtype the wall safe, or use a floor/full tile safe.
<br>
<img width="956" height="901" alt="image" src="https://github.com/user-attachments/assets/d2e5d8ca-ecea-4d8e-9497-21b3c6d61561" />
Turn this area into a full on meeting space, rather than having it be this kinda empty hallway. Give them a like, table in the middle. some chairs or benches. There's potential here.
<br>
<img width="541" height="466" alt="image" src="https://github.com/user-attachments/assets/1fd428d1-5f70-4564-a69a-1e00b87dc63f" />
Cargo space here needs more decalling. Add some boxes under the crate shelves or something
<br>
<img width="458" height="205" alt="image" src="https://github.com/user-attachments/assets/55246bc2-d6da-46e1-8fe4-fd6f381de3cb" />
missing a pipe
<br>
<img width="308" height="328" alt="image" src="https://github.com/user-attachments/assets/c9e3f678-27f6-406b-8b07-595196aa2105" />
Please pixelshift locker gear so you get a good idea of whats in it at a glance
<br>
<img width="1154" height="868" alt="image" src="https://github.com/user-attachments/assets/6eeec2da-3eeb-4a0b-b706-cd9e33285b89" />
less of this space needs to be hallway. Consider expanding the engineering area to give the engineer role a proper little office.
<br>
<img width="238" height="215" alt="image" src="https://github.com/user-attachments/assets/e409f7c3-63e4-4fd9-b62a-11a6a62f0d16" />
placement of the autolathe is fairly arbitrary, consider again moving it.
<br>
<img width="1015" height="1253" alt="image" src="https://github.com/user-attachments/assets/a8158a3b-1c63-4404-a190-5db93b37887e" />
go for a less technically decalled style here. Try miming the harrier, with more added red. I do like the telecomms floors, keep them under the computers?

<img width="438" height="540" alt="image" src="https://github.com/user-attachments/assets/4ed2d478-f594-4a4e-bbc4-787184299ede" />
Add radio closet to cryo

<img width="542" height="769" alt="image" src="https://github.com/user-attachments/assets/cfc870d5-8572-477b-9b3e-bd389f34f35d" />
visiting back here. try turning it into more of a lounge, rather than two chairs against a table. maybe a small sofa.. Also add in some decorative food on the tables. Helps the lived in feel.

Burning02 commented:
Should be nearly every review addressed, just need to make a wall safe that inherits whatever is on the ground, and the meeting area you talked about

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 commented:
Also updated the Strongdmm photos, just need a few moments for the ingame photo

Burning02 committed:
Update _maps/configs/nanotrasen_shrike.json

Co-authored-by: Erika Fox <94164348+Erikafox@users.noreply.github.com>
Signed-off-by: Burning02 <141873051+Burning02@users.noreply.github.com>

Burning02 committed:
Safe

Burning02 committed:
Merge branch 'Shriked-up' of https://github.com/Burning02/Shiptest into Shriked-up

Burning02 committed:
Finally fully addressed

Gristlebee reviewed (dismissed):
Please fix your linter checks. If you're capitalizing names, the shutters should have the first letter of each word be capitalized. 

Eg. Cargo **B**ay instead of Cargo **b**ay
<img width="2003" height="830" alt="image" src="https://github.com/user-attachments/assets/85cf1ac3-59eb-42c3-a3fc-bc3ef715c7aa" />
<img width="566" height="42" alt="image" src="https://github.com/user-attachments/assets/6a057696-c055-4422-8a33-63886ad70515" />
<img width="562" height="39" alt="image" src="https://github.com/user-attachments/assets/0787fee3-928b-48f0-b9a2-8439eb24bcab" />
<img width="557" height="32" alt="image" src="https://github.com/user-attachments/assets/6fca4f6d-0985-4c96-9403-bd061cce163f" />

Burning02 commented:
There we go, finalized the last few things I hadn't done yet, review _should_ be fully addressed now?

Burning02 commented:
Nevermind, I just noticed the new one,hold on

Burning02 committed:
The names....

Burning02 commented:
Ok now its all (Probably) addressed

Burning02 committed:
Merge branch 'master' into Shriked-up

Burning02 committed:
Curse you wheel of cheese (And misc changes)

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Whoops

Didn't fix my SMES output when I thought I did

procdrone commented:
Sergeant does not have bridge access.
in Sergeant quarters, the locker name is "lieutenant's locker"

Burning02 commented:
> Sergeant does not have bridge access. in Sergeant quarters, the locker name is "lieutenant's locker"

Forgot to change the locker name when deciding on the rank, but the access problem is odd, since I made them a subtype of the HoS if I recall. I'll see what I can do about that when able

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Merge conflicts? Never heard of her

Burning02 committed:
Whoops

Burning02 committed:
Update nanotrasen.dm

Burning02 committed:
Merge branch 'master' into Shriked-up

Burning02 committed:
Update nanotrasen.dm

Burning02 committed:
Update nanotrasen.dm

Burning02 committed:
Update nanotrasen.dm

Burning02 commented:
As long as none of the checks fail, fixed the merge conflict, and did the few changes requested by the people who first played it

Burning02 commented:
Updated the Strong photos, but can't currently update the ingame one

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 commented:
There we go, pushed the recent changes and ready for another review

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 committed:
Merge branch 'master' into Shriked-up

Burning02 committed:
Whoops

Burning02 committed:
Merge branch 'Shriked-up' of https://github.com/Burning02/Shiptest into Shriked-up

thearbiber commented:
Not seeing anything glaring. Should be good for TM again.

Luftkommando commented:
So a couple things from my playtest on 13/08 

The security office window has no floor under it and is actually just straight up space, even on the outpost.
<img width="209" height="207" alt="dreamseeker_uIsUafB3ye" src="https://github.com/user-attachments/assets/c2500f41-aacf-4f21-987f-c06bb459ea14" />

Supposedly there's a missing wire here? I didn't catch this one myself, it was the mechanic I had on shift.
<img width="39" height="34" alt="firefox_8m0ZlVtr5f" src="https://github.com/user-attachments/assets/31d59917-eb3a-40ad-9f8e-0fdcf064e2f1" />

I didn't catch anything else before my keyboard went haywire and I had to resort to using the digital keyboard, which slowed me down considerably.

Burning02 committed:
Update nanotrasen_shrike.dmm

Burning02 commented:
All the above problems should be fixed now

Burning02 committed:
I love piping

Just changing the layer of the plasma thrusters from 1 to 3, so in the off-chance you blow up your ship you can repair the afterburners

Burning02 committed:
Merge branch 'master' into Shriked-up
